### PR TITLE
Death to INOPs

### PR DESCRIPTION
**Pull request by J-Hoskin**
This merge merges over many of the features in my CJ4-Extended Repo, plus a few extra cockpit features.

### CODE REVIEW

**Comment by mattnischan** [comment]
@J-Hoskin,

Definitely appreciate the work and contribution! I think as a general comment, we have tried to edit as few globally used files as possible, to make this mod as maximally compatible with any other aircraft. As this is structured at the moment, it touches a lot of files that are used in multiple planes.

I think before accepting, we would want those changes pulled up into the CJ4 cockpit file so that the changes are guaranteed to be isolated to a single aircraft, and not conflict with any other mods of those shared files.

**Comment by J-Hoskin** [comment]
@mattnischan Okay, fair enough. I'll see what I can do.

**Comment by mattnischan** [comment]
@J-Hoskin, feel free to leave this open as you work on it, if you like. The PR will automatically pick up changes you make in your branch in your fork as you go.

**Comment by J-Hoskin** [comment]
@mattnischan I'll let you know when its ready.

**Comment by mattnischan** [comment]
Sounds good!

**Comment by J-Hoskin** [comment]
Okay, I have encapsulated the xml files into the CJ4 directory. I kept the same subtemplate architecture cause I think its best but it is separate to all other add-ons.

File: workingtitle-project-cj4.xml [hunk 1]
```diff
@@ -5,5 +5,6 @@
 		<Package>PackageDefinitions\workingtitle-vcockpits-instruments-cj4.xml</Package>
 		<Package>PackageDefinitions\workingtitle-vcockpits-instruments-airliners.xml</Package>
 		<Package>PackageDefinitions\workingtitle-aircraft-cj4.xml</Package>
+		<Package>PackageDefinitions\workingtitle-base-aircraft-common.xml</Package>
```
Comment (mattnischan) [inline workingtitle-project-cj4.xml:8]: It looks like you pulled this package, but maybe left this line still here by accident.

**Review by mattnischan — COMMENTED** [review]
Looks good! I think my only additional general comment is on the folder structure: we haven't used `CJ4_Extended` anywhere else in the project, so that might be a bit confusing for devs, thinking maybe it's a different addon to these mods.

**Comment by mattnischan** [comment]
@J-Hoskin One more question,

With the way this is structured, if Asobo fixes things in the core ModelDefs for other buttons and switches we haven't modified so far, will we receive those updates or does this pull all the definitions into our space?

File: workingtitle-project-cj4.xml (hunk 1 above)
Comment (J-Hoskin) [reply workingtitle-project-cj4.xml:8]: Ok, I'll render these points.

**Comment by J-Hoskin** [comment]
> @J-Hoskin One more question,
> 
> With the way this is structured, if Asobo fixes things in the core ModelDefs for other buttons and switches we haven't modified so far, will we receive those updates or does this pull all the definitions into our space?

The files I added only overwrite the buttons which use those new templates in the Cockpit.xml. So if Asobo push changes for aircon knob functionality, our functionality will replace those new changes. If they add functionality to say the the  ELT which isn't using our new templates, then those new changes will take affect.

**Comment by J-Hoskin** [comment]
@mattnischan this is perfected.

**Review by mattnischan — APPROVED** [review]

**Comment by cwburnett** [comment]
@J-Hoskin are you still working on this one, or are the sounds you're working on for aircon and things going to be in a separate PR?

**Comment by J-Hoskin** [comment]
@cwburnett @mattnischan yeh its good to go. There is some small sound bugs where if you switch to exterior view and but to interior, the air fan sounds stop. But imo thats fine enough for now.

**Comment by J-Hoskin** [comment]
**CHANGE LOG:**
- Added functionality to air and heat knob (Pilot, Copilot, and Cabin Fan, Cockpit and Cabin Temperature, Climate Control, Source)
- Added seatbelt announcement sound and seatbelt light functionality
- Added sounds to the air fans

**Review by dga711 — APPROVED** [review]

**Review by cwburnett — APPROVED** [review]

**Pull request merged**